### DISCUSSION TITLE
Configuration Files

### DISCUSSION THREAD

**Discussion by sergiosilva-nsj** [Erudika/scoold#313] [Q&A] [answered]
Can I use environment variables instead of configuration files?

Because it is more easy to change, when I am configuring the environment though docker images (in Kubernets, for example).

Thanks!

**Comment by albogdano** [comment, marked as answer]
Yes, of course - you can set environment variables with underscores instead of dots. For example `scoold.port = 8000` becomes `scoold_port=8000`. Also you can set the environment variable `JAVA_OPTS="-Dscoold.port=8000 -Dscoold.env=production"`.

**Reply by sergiosilva-nsj** [reply]
Thanks!
